compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -1587,13 +1590,31 @@ LLVMRustThinLTOBufferCreate(LLVMModuleRef M, bool is_thin) {
   {
     raw_string_ostream OS(Ret->data);
     {
-      legacy::PassManager PM;
       if (is_thin) {
+#if LLVM_VERSION_LT(16, 0)
```

Comment:
Can we go ahead and use newPM on LLVM 15 too? It seems to work fine for me, and we'll get more continuous testing that way. Plus, in other contexts, `should_use_new_llvm_pass_manager` is already forcing `true` for LLVM >= 15.